### PR TITLE
[TW-953] Doc Improved UX for debugging test failures

### PR DESCRIPTION
**Pull request by mark-anthony-postman**

### CODE REVIEW

File: src/pages/docs/running-collections/intro-to-collection-runs.md [hunk 1]
```diff
@@ -70,51 +65,48 @@ You can configure the Collection Runner to meet your development needs. You can
     * **Iterations** - The number of iterations for your collection run. You can also run collections multiple times with different data sets to [build workflows](/docs/running-collections/building-workflows/).
     * **Delay** - An interval delay in milliseconds between each request.
     * **Data** - A [data file](/docs/running-collections/working-with-data-files/) for the collection run.
+    * **Persist responses for a session** - Log the response headers and bodies so you can review them after running the collection. For large collections, persisting responses may affect performance.
+
+      > Request and response details are persisted for your current Postman session and aren't saved permanently. Signing out of Postman, signing into another device with the same account, or refreshing your browser will end your session and remove the logged data.
```
Comment (postfrzero) [inline src/pages/docs/running-collections/intro-to-collection-runs.md:70]: I wonder if we would want to explicitly mention that the request/response data doesn't leave their machine? It's implied by "session", I suppose, but sessions can technically be persisted across the server as well. 🤔

Something like this perhaps:

> Request and response details are persisted locally during your current Postman session and aren't saved permanently.

**Review by postfrzero — APPROVED** [review]
Looks good to me! I've just left one comment, but it's more of a lingering thought.

File: src/pages/docs/running-collections/intro-to-collection-runs.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/running-collections/intro-to-collection-runs.md]: @postfrzero Yes, I think it's a great idea to clarify that the data is persisted locally. I've added your suggested wording. Thanks! 😅

File: src/pages/docs/running-collections/intro-to-collection-runs.md [hunk 2]
```diff
@@ -70,51 +65,48 @@ You can configure the Collection Runner to meet your development needs. You can
...
 
-1. To view more details, including any log statements used in your collection, select <img alt="Console icon" src="https://assets.postman.com/postman-docs/icon-console-v9.jpg#icon" width="16px"> __Console__ from the footer.
+* Select a request to view details about the request. You can view general information about the request and the request headers and body. You can also view the response headers and body if you selected the **Persist responses for a session** option when [configuring the collection run](#configuring-a-collection-run).
 
-1. To return to the summary of the run, select __View Results__.
+* Hover over a request and select <img alt="External link icon" src="https://assets.postman.com/postman-docs/icon-external-link.jpg#icon" width="18px"> **Open Request** to open the request in a new tab. You can view any test scripts or select **Send** to send the request again.
 
-### Viewing test results
+* Select the **Passed**, **Failed**, or **Skipped** tabs to filter the results by test status. To show all requests, select the **All Tests** tab. If any test in a request script fail during a collection run, the whole request fails.
```
Comment (petewitcher-postman) [inline src/pages/docs/running-collections/intro-to-collection-runs.md:101]:
```suggestion
* Select the **Passed**, **Failed**, or **Skipped** tabs to filter the results by test status. To show all requests, select the **All Tests** tab. If any tests in a request script fail during a collection run, the whole request fails.
```

**Review by petewitcher-postman — APPROVED** [review]
LGTM! Found one typo.

File: src/pages/docs/running-collections/intro-to-collection-runs.md [hunk 3]
```diff
@@ -70,51 +65,48 @@ You can configure the Collection Runner to meet your development needs. You can
...
 
 1. When you've completed your configuration, select **Run (collection name)**.
 
+![Collection Runner configuration](https://assets.postman.com/postman-docs/v10/collection-run-configuration-v10-12.jpg)
+
 > Your [Postman plan](https://www.postman.com/pricing/) gives you a limited number of collection runs you can use each month. This limit applies to collections that you run in a workspace or the Scratch Pad using the **Run manually** option. This limit doesn't apply to [scheduled collection runs](/docs/running-collections/scheduling-collection-runs/) in the Postman cloud. A collection run with multiple iterations counts as a single run.
 >
 > A message will display in the Collection Runner when you're approaching your usage limit. Learn more about [resource usage](/docs/administration/resource-usage/) in Postman.
 
 ## Running your collections
```
Comment (malvikach) [inline src/pages/docs/running-collections/intro-to-collection-runs.md:89]: I think this heading can be changed to  "Debugging Run Results"

Comment (mark-anthony-postman) [reply src/pages/docs/running-collections/intro-to-collection-runs.md]: Thanks! I updated the heading as you suggested 😀

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**